static/app/views/dashboards/widgetBuilder/buildSteps/yAxisStep/yAxisSelector/index.tsx

```diff
@@ -136,7 +177,15 @@ export function YAxisSelector({
 
       {!hideAddYAxisButtons && (
         <Actions gap={1}>
-          <AddButton title={t('Add Overlay')} onAdd={handleAddOverlay} />
+          <AddButton
+            title={
+              displayType === DisplayType.BIG_NUMBER &&
+              organization.features.includes('dashboards-bignumber-equations')
+                ? t('Add Field')
```

Comment:
We don't have to change it in this PR, but maybe "Add Aggregate"? "Add Function"? We may need to ask Ale about the wording we like to present here